my_scratch_file.py

```diff
@@ -0,0 +1,20 @@
+import turtle
```

Comment:
Normally scratch files shouldn't be added and pushed to the repository. Removing it isn't straightforward though, so we will leave it for now. In the future, only commit and push the files that are part of the assignment.